frontend/packages/gitops-plugin/integration-tests/features/gitops/linkArgoCD.feature

```diff
@@ -0,0 +1,26 @@
+Feature: External Argo CD Link for Argo CD Dashboard
+              As a developer, I want a way to navigate to Argo CD from the Deploy Stages page of the OpenShift console so that I can get insight on the sync status of my application.
+
+
+        Background:
+            Given user is at the Administrator perspective
+              And user has installed Sealed Secrets Operator
+              And user has created SealedSecretController CR
+              And user has installed OpenShift Pipelines Operator
+              And user has installed Argo CD Operator
```

Comment:
We do not need these steps for gitops operator installation. Gitops operator install argocd and pipeline operator bundled with it. As part of sealed secret things are concerned, it is required for kam bootstrap command.

Reply:
ok, I will remove these lines from the file

Reply:
updated